### PR TITLE
fixed the s3 endpoint in the vpc module

### PR DESCRIPTION
**Pull request by tgallagher91**

### CODE REVIEW

**Comment by tgallagher91** [comment]
this actually fixes the issue

File: modules/aws/vpc/main.tf [hunk 1]
```diff
@@ -194,14 +194,19 @@ resource "aws_route" "workspaces_default_route_fw" {
   route_table_id         = element(aws_route_table.workspaces_route_table.*.id, count.index)
 }
 
-data "aws_vpc_endpoint_service" "s3" {
-  service = "s3"
+ddata "aws_vpc_endpoint_service" "s3" {
```
Comment (jakeasarus) [inline modules/aws/vpc/main.tf:197]: misspelled data

**Review by jakeasarus — CHANGES_REQUESTED** [review]
Need to fix misspelling of 'data' on line 197, currently says 'ddata'

File: modules/aws/vpc/main.tf (hunk 1 above)
Comment (tgallagher91) [reply modules/aws/vpc/main.tf]: i have no idea how the code worked with DATA MISPELLED

**Review by jakeasarus — COMMENTED** [review]
looks better. @tgallagher91 also tested the module before pulling in to verify that broken runs are now successful.

**Review by jakeasarus — APPROVED** [review]
lgtm

**Comment by jakeasarus** [comment]
made minor syntax updates to remove count. Know working with this run: https://app.terraform.io/app/thinkstack-co/workspaces/peoples_prod_uswest2_infrastructure/runs/run-znhs7phqfpYx5XqV

ready for approval.

File: modules/thinkstack/siem/outputs-Jacob’s MacBook Pro.tf [hunk 2]
```diff
@@ -0,0 +1,125 @@
+output "private_subnet_ids" {
```
Comment (zachreborn) [inline modules/thinkstack/siem/outputs-Jacob’s MacBook Pro.tf:1]: Any reason we changed the outputs file name? @jakeasarus

**Review by zachreborn — CHANGES_REQUESTED** [review]
Can you rename back to outputs.tf please?

File: modules/thinkstack/siem/outputs-Jacob’s MacBook Pro.tf (hunk 2 above)
Comment (jakeasarus) [reply modules/thinkstack/siem/outputs-Jacob’s MacBook Pro.tf]: mistake. fixed!

**Review by zachreborn — APPROVED** [review]

**Pull request merged**